.gitignore

```diff
@@ -167,6 +167,7 @@ test.sqlite
 user_datasets
 user_runs
 user_models
+user_explainers
```

Comment:
```suggestion
```

With the new location of the explanations in the local folders, it should no longer be a problem to add these ignores.